spec/support/core_helpers.rb

```diff
@@ -43,6 +47,19 @@ module CoreHelpers
     def supports_value_expectations?
       false
     end
+
+    # Add support for `once`, `at_least`, and other chained matchers from `RSpec::Mocks::Matchers::Receive`
```

Comment:
This is used to allow for `expect { subject }.to log_deprecation('deprecated_option').at_least(1).time`.